Getting_Ready_to_Chess.cpp

```diff
@@ -79,5 +89,22 @@ int main()
             }
         }
     }
+}
+
+int main()
+{
+    std::string nameFile = "Koordinates.txt";
+    std::vector<Figure*> figure_array;
```

Comment:
Don't forget that figure_array holds pointers to dynamic memory obtained with new operator. How to properly clean memory in the end?